src/components/application_manager/test/policy_handler_test.cc

```diff
@@ -3031,10 +3032,33 @@ TEST_F(PolicyHandlerTest, GetAppPropertiesStatus_NicknameChanged_SUCCESS) {
             policy_handler_.GetAppPropertiesStatus(properties, kPolicyAppId_));
 }
 
-TEST_F(PolicyHandlerTest, GetAppPropertiesStatus_NicknameNotChanged_SUCCESS) {
+TEST_F(PolicyHandlerTest,
+       GetAppPropertiesStatus_NicknameRemoveChanged_SUCCESS) {
   ChangePolicyManagerToMock();
 
-  const std::string kFakeNickname = "fake_nickname";
+  smart_objects::SmartObject properties;
+  properties[strings::app_id] = kPolicyAppId_;
+  properties[strings::nicknames] =
+      smart_objects::SmartObject(smart_objects::SmartType_Array);
+  properties[strings::nicknames].asArray()->push_back(
+      smart_objects::SmartObject(kFakeNickname));
+
+  std::shared_ptr<policy::StringArray> nicknames =
```

Comment:
```suggestion
  auto nicknames =
```

Reply:
@mvorobio 
Thanks,fixed in:https://github.com/smartdevicelink/sdl_core/pull/3336/commits/e8b6796277ec4c559a26fb8e02562d0929e67b94